rust/main/Cargo.toml

```diff
@@ -256,42 +236,42 @@ version = "=0.5.2"
 
 [patch.crates-io.solana-account-decoder]
 git = "https://github.com/hyperlane-xyz/solana.git"
-tag = "hyperlane-1.14.13-2023-07-04"
+branch = "ameten/hyperlane-1.14.13-compilable"
```

Comment:
I assume after https://github.com/hyperlane-xyz/solana/pull/2 is merged you'll create a tag there instead of relying on the branch?

Reply:
ah yes I see your discord message about this

Reply:
It is correct.